packages/cli-v3/src/entryPoints/managed/execution.ts

```diff
@@ -943,14 +863,40 @@ export class RunExecution {
       properties: {
         ...properties,
         runId: this.runFriendlyId,
-        snapshotId: this.currentSnapshotId,
+        snapshotId: this.currentSnapshotFriendlyId,
         executionId: this.id,
         executionRestoreCount: this.restoreCount,
         lastHeartbeat: this.lastHeartbeat?.toISOString(),
       },
     });
   }
 
+  private sendRuntimeDebugLog(
+    message: string,
+    properties?: SendDebugLogOptions["properties"],
+    runIdOverride?: string
+  ) {
+    this.logger.sendDebugLog({
+      runId: runIdOverride ?? this.runFriendlyId,
+      message: `[runtime] ${message}`,
+      print: false,
+      properties: {
+        ...properties,
+        runId: this.runFriendlyId,
+        snapshotId: this.currentSnapshotFriendlyId,
+        executionId: this.id,
+        executionRestoreCount: this.restoreCount,
+        lastHeartbeat: this.lastHeartbeat?.toISOString(),
+      },
+    });
+  }
+
+  private set suspendable(suspendable: boolean) {
```

Comment:
If this is called and it does fail (the catch fires), should we return false from the function?